primitives/account/tests/accounts.rs

```diff
@@ -833,10 +834,10 @@ fn it_commits_valid_and_failing_txns() {
         1,
         NetworkId::Dummy,
     );
-    tx.sender_type = AccountType::Basic;
+    // tx.sender_type = AccountType::Basic;
```

Comment:
Why did you comment this?

Reply:
I think because it's technically not needed, as `Basic` is the default.